alma/upgrade/upgrade-2.6.0.php

```diff
@@ -54,6 +55,7 @@ function upgrade_module_2_6_0()
 
             CustomFieldsHelper::initCustomFields();
         } catch (RequestError $e) {
+            Logger::instance()->error("[Alma] ERROR upgrade v2.6.0: {$e->getMessage()}");
```

Comment:
Are you sure you want return `true` if an Exception occurs ?

Reply:
@syjust-alma 
It was handled like that before, I took it back as it was. I guess it's to not block the update of the module.